docker-compose.yml

```diff
@@ -411,6 +411,7 @@ services:
       - WIKIBASE_SCHEME=${WIKIBASE_SCHEME:-https}
       - WDQS_HOST=wdqs.svc
       - WDQS_PORT=9999
+      - BLAZEGRAPH_OPTS='-Dorg.wikidata.query.rdf.tool.rdf.RdfRepository.timeout=3600'
```

Comment:
@gehel with this change, we were unsuccessfully trying to increase the timeout of for queries for our blazegraph instance. I did found your contact on https://www.mediawiki.org/wiki/Wikidata_Query_Service/Implementation but the setting above was pretty much the only hint to successfull extension of the timeout I was able to find. Do you happen to know if there is any documentation if I (in the role of the most stupid sys-admin) would want to increase the query timeout.

Currently, the error we see is 
```
16:11:16.254 [qtp584634336-29] ERROR c.b.r.sail.webapp.BigdataRDFServlet IP:docker-wdqs-proxy-1.docker_default UA:Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0 - cause=java.util.concurrent.TimeoutException, query=SPARQL-QUERY: queryStr=PREFIX wdt: <https://portal.mardi4nfdi.de/prop/direct/>

PREFIX wd: <https://portal.mardi4nfdi.de/entity/>

SELECT ?item ?property_a_value

WHERE {

?item wdt:P31 wd:Q56887.

?item wdt:P1451 ?property_a_value.

}

16:11:16.267 [com.bigdata.journal.Journal.executorService26] WARN com.bigdata.util.concurrent.Haltable IP: UA: - com.bigdata.util.concurrent.Haltable@1dff8040 : isFirstCause=true : com.bigdata.bop.engine.QueryTimeoutException: Query deadline is expired.
```

PS: Unfortunately, the UI on our server https://sparql.portal.mardi4nfdi.de/ vanish after updating to version 0.6, however, one can execute stuff by putting queries in the url like
https://sparql.portal.mardi4nfdi.de/bigdata/namespace/wdq/sparql?query=PREFIX%20wdt%3A%20%3Chttps%3A%2F%2Fportal.mardi4nfdi.de%2Fprop%2Fdirect%2F%3E%0APREFIX%20wd%3A%20%3Chttps%3A%2F%2Fportal.mardi4nfdi.de%2Fentity%2F%3E%0A%0ASELECT%20%3Fitem%20%3Fproperty_a_value%0AWHERE%20%7B%0A%3Fitem%20wdt%3AP31%20wd%3AQ56887.%0A%3Fitem%20wdt%3AP1451%20%3Fproperty_a_value.%0A%7D (downloads 300MB)